projects/opendr_ws/src/opendr_perception/scripts/fall_detection_node.py

```diff
@@ -18,65 +18,72 @@
 import torch
 
 import rospy
-from vision_msgs.msg import Detection2DArray
+from vision_msgs.msg import Detection2D
 from sensor_msgs.msg import Image as ROS_Image
 from opendr_bridge import ROSBridge
+from opendr_bridge.msg import OpenDRPose2D
 
 from opendr.engine.data import Image
-from opendr.engine.target import BoundingBox, BoundingBoxList
+from opendr.engine.target import BoundingBox
 from opendr.perception.pose_estimation import get_bbox
 from opendr.perception.pose_estimation import LightweightOpenPoseLearner
 from opendr.perception.fall_detection import FallDetectorLearner
 
 
 class FallDetectionNode:
 
-    def __init__(self, input_rgb_image_topic="/usb_cam/image_raw",
+    def __init__(self, input_rgb_image_topic="/usb_cam/image_raw", input_pose_topic="/opendr/poses",
```

Comment:
The `input_rgb_image_topic` should be `None` by default?
If I'm not wrong this is what is stated in the documentation.